client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx

```diff
@@ -1,53 +1,60 @@
 import React from 'react'
-import { fireEvent, waitFor, render, screen } from '@testing-library/react'
-import { BrowserRouter as Router, useParams } from 'react-router-dom'
+import userEvent from '@testing-library/user-event'
+import { screen, fireEvent, waitFor } from '@testing-library/react'
+import { useParams } from 'react-router-dom'
 import { toast } from 'react-toastify'
 import relativeStages from '../_mocks'
 import Review from './index'
 import * as utilities from '../../../utilities'
 import useAuditSettings from '../../useAuditSettings'
 import { settingsMock, sampleSizeMock } from './_mocks'
-import { IJurisdiction } from '../../useJurisdictions'
 import { contestMocks } from '../Contests/_mocks'
-import { ISampleSizeOption, IContest } from '../../../../types'
 import { jurisdictionMocks } from '../../_mocks'
+import { withMockFetch, renderWithRouter } from '../../../testUtilities'
+import { ISampleSizes } from './useSampleSizes'
 
 const auditSettingsMock = useAuditSettings as jest.Mock
 
-const apiMock: jest.SpyInstance<
-  ReturnType<typeof utilities.api>,
-  Parameters<typeof utilities.api>
-> = jest.spyOn(utilities, 'api').mockImplementation()
-
-const generateApiMock = (
-  sampleSizeReturn: { sampleSizes: ISampleSizeOption[] },
-  contestsReturn: { contests: IContest[] } | Error | { status: 'ok' },
-  jurisdictionReturn:
-    | { jurisdictions: IJurisdiction[] }
-    | Error
-    | { status: 'ok' }
-) => async (endpoint: string) => {
-  switch (endpoint) {
-    case '/election/1/sample-sizes':
-      return sampleSizeReturn
-    case '/election/1/jurisdiction':
-      return jurisdictionReturn
-    case '/election/1/jurisdiction/file':
-      return { file: null, processing: { status: 'PROCESSED' } }
-    case '/election/1/contest':
-      return contestsReturn
-    case '/election/1/round':
-    default:
-      return { status: 'ok' }
-  }
+const apiCalls = {
+  getSampleSizeOptions: {
+    url: '/api/election/1/sample-sizes',
+    response: sampleSizeMock,
+  },
+  postRound: (sampleSizes: ISampleSizes) => ({
+    url: '/api/election/1/round',
+    response: { status: 'ok' },
+    options: {
+      body: JSON.stringify({
+        sampleSizes,
+        roundNum: 1,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      method: 'POST',
+    },
+  }),
+  getEmptyJurisdictions: {
+    url: '/api/election/1/jurisdiction',
+    response: { jurisdictions: [] },
+  },
+  getJurisdictions: {
+    url: '/api/election/1/jurisdiction',
+    response: { jurisdictions: jurisdictionMocks.allManifests },
+  },
```

Comment:
could combine these like you did with `postRound`. same with the `getContests` calls below